app/authenticate.py

```diff
@@ -0,0 +1,34 @@
+from flask_httpauth import HTTPTokenAuth, HTTPBasicAuth, MultiAuth
+from flask import jsonify, g
+
+from app.models import User
+
+token_auth = HTTPTokenAuth("Bearer")
+passw_auth = HTTPBasicAuth()
```

Comment:
I decided to remove password based authentication, the application handles all security via tokens.